src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -27,6 +27,13 @@ export type PiaStateChangeHandlerType = (
 
 export type PiaFormOpenMode = 'edit' | 'view';
 
+export interface PiaValidationMessage {
+  piaTitle?: string | null;
+  piaMinistry?: string | null;
+  piaBranch?: string | null;
+  piaInitialDescription?: string | null;
+}
+
```

Comment:
shouldn't these validation messages be inside intake form component and not the parent component?

Reply:
will create a refactor ticket for this